packages/cli/__mocks__/fs.js

```diff
@@ -0,0 +1,34 @@
+import { isString } from 'lodash'
+
+const fs = {
+  ...require.requireActual('fs'),
+}
```

Comment:
I tried this instead:
```js
const fs = jest.genMockFromModule('fs')
```
But it breaks in a weird way:
![image](https://user-images.githubusercontent.com/182853/81564925-9104fa00-93a9-11ea-876d-719cafdeff0f.png)